src/blueprints/tv2/value-objects/tv2-video-mixer-transition.ts

```diff
@@ -0,0 +1,28 @@
+export enum VideoMixerTransition {
+  MIX = 'MIX',
+  DIP = 'DIP',
+  WIPE = 'WIPE',
+  SPLIT_SCREEN = 'SPLIT_SCREEN',
+  STING = 'STING',
+  CUT = 'CUT',
+  DUMMY = 'DUMMY'
+}
+
+export interface VideoMixerTransitionSettings {
+  mix?: {
```

Comment:
It would be nice if `mix` and `dip` had their `rate` attributes renamed to `frameRate` as it was done for `wipe`.

Reply:
has been renamed as requested.